blog-cse/2023-03-24-content.md

```diff
@@ -0,0 +1,34 @@
+---
+title: March 24, 2023 - Content Release
+hide_table_of_contents: true
+image: https://help.sumologic.com/img/sumo-square.png
+authors:
+  - url: https://help.sumologic.com/release-notes-cse/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+Overall improvements to OOTB First Seen rules include, minor baseline tweaks and severity adjustments for the following rules.
+For corrections involving logic adjustment, additional context is included within the individual rule. Added Alternative Values for ProofPoint TAP Mappers.
```

Comment:
I'm not fond of the last sentence in this line since it's an incomplete sentence and I'm not sure what you men. Could you write this as a complete sentence?
"Added Alternative Values for ProofPoint TAP Mappers."

Reply:
I see the fix in commit [Update 2023-03-24-content.md](https://github.com/SumoLogic/sumologic-documentation/pull/1949/commits/743a7fbeda013c498d38375700b1e6dd251b13c6)